django/db/models/fields/__init__.py

```diff
@@ -1500,6 +1500,8 @@ def __init__(self, verbose_name=None, name=None, max_digits=None,
                  decimal_places=None, **kwargs):
         self.max_digits, self.decimal_places = max_digits, decimal_places
         super(DecimalField, self).__init__(verbose_name, name, **kwargs)
+        self.validators.append(validators.DecimalValidator(self.max_digits,
+                                                           self.decimal_places))
```

Comment:
The system check framework will throw an error if a model uses `DecimalField` without those arguments. I'm not sure if there's a use case for instantiating `DecimalField` outside of a model (perhaps in `output_field`, but don't think validators are used there). All in all, I think the current version is okay.